webapp/src/ts/reducers/global.ts

```diff
@@ -23,6 +23,7 @@ const initialState = {
   },
   facilities: [],
   filters: {},
+  sidebarFilter: {},
```

Comment:
The `sidebarFilter` keeps the component state that helps interoperability with other components outside the scope.
The `filter` keeps the actual selected criteria to filter data.